plugins/wme-sitebuilder/wme-sitebuilder/Concerns/HasWordPressDependencies.php

```diff
@@ -76,22 +76,22 @@ public function getPluginVersion( $plugin_file ) {
 	}
 
 	/**
-	 * Verify that plugin is installed and the version is supported.
+	 * Verify that plugin is installed and the version is within a supported range.
 	 *
-	 * @uses $this->getPluginVersion()
-	 *
-	 * @param string $plugin           The directory/file path.
-	 * @param string $expected_version The version to check against.
+	 * @param  string  $plugin       The directory/file path.
+	 * @param  string  $min_version  The minimum plugin version supported.
+	 * @param  string  $max_version  The maximum plugin version supported.
 	 *
 	 * @return bool
 	 */
-	public function isPluginVersion( $plugin, $expected_version ) {
+	public function isPluginInVersionRange( $plugin, $min_version, $max_version ) {
```

Comment:
Would this make more sense as `isPluginVersionInRange`?

Reply:
Great catch, thanks. Fixed in https://github.com/moderntribe/wme/pull/204/commits/bc96df91a7d432e0cba37514e46bd21e9909eed4